instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -197,6 +209,7 @@ def process_request(self, request):
 
         is_asgi_request = _is_asgi_request(request)
         if not _is_asgi_supported and is_asgi_request:
+            _OnceLogger.log_warning_asgi_instrumentation()
```

Comment:
Please add a test case, see [this](https://github.com/open-telemetry/opentelemetry-python/pull/3360/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R74-R84).

Reply:
Is there already a test for a pattern where the request type is `ASGIRequest` and `opentelemetry-instrumentation-asgi` is not installed?